### PR TITLE
fix: Refresh task for Items with corrupted meta

### PR DESCRIPTION
**Pull request by yakischik**

### CODE REVIEW

**Review by dmitry-saraykin — APPROVED** [review]

File: worker/src/main/kotlin/com/rarible/protocol/union/worker/task/meta/RefreshMetaWithCorruptedUrlTask.kt [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    private val itemRepository: ItemRepository,
+    private val itemMetaService: ItemMetaService
+) {
+
+    private val logger = LoggerFactory.getLogger(javaClass)
+
+    fun handle(continuation: String?, param: String): Flow<String> {
+        val from = continuation?.let { ShortItemId.of(it) }
+        val pattern = Pattern.compile(param)
+        return itemRepository.findAll(from).map {
```
Comment (sanufriev) [inline worker/src/main/kotlin/com/rarible/protocol/union/worker/task/meta/RefreshMetaWithCorruptedUrlTask.kt:43]: Don't it be too long?

Comment (sanufriev) [reply worker/src/main/kotlin/com/rarible/protocol/union/worker/task/meta/RefreshMetaWithCorruptedUrlTask.kt:43]: Can we make batch handle?

File: worker/src/main/kotlin/com/rarible/protocol/union/worker/task/meta/RefreshMetaWithCorruptedUrlTask.kt [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+@Component
+class RefreshMetaWithCorruptedUrlJob(
+    private val itemRepository: ItemRepository,
+    private val itemMetaService: ItemMetaService
+) {
+
+    private val logger = LoggerFactory.getLogger(javaClass)
+
+    fun handle(continuation: String?, param: String): Flow<String> {
```
Comment (sanufriev) [inline worker/src/main/kotlin/com/rarible/protocol/union/worker/task/meta/RefreshMetaWithCorruptedUrlTask.kt:40]: What is "param" - what reg example? how is using with job?

File: worker/src/main/kotlin/com/rarible/protocol/union/worker/task/meta/RefreshMetaWithCorruptedUrlTask.kt (hunk 1 above)
Comment (yakischik) [reply worker/src/main/kotlin/com/rarible/protocol/union/worker/task/meta/RefreshMetaWithCorruptedUrlTask.kt:43]: Its just pattern check, batch handling wont help here

File: worker/src/main/kotlin/com/rarible/protocol/union/worker/task/meta/RefreshMetaWithCorruptedUrlTask.kt (hunk 2 above)
Comment (yakischik) [reply worker/src/main/kotlin/com/rarible/protocol/union/worker/task/meta/RefreshMetaWithCorruptedUrlTask.kt:40]: See line 28

File: worker/src/main/kotlin/com/rarible/protocol/union/worker/task/meta/RefreshMetaWithCorruptedUrlTask.kt (hunk 1 above)
Comment (sanufriev) [reply worker/src/main/kotlin/com/rarible/protocol/union/worker/task/meta/RefreshMetaWithCorruptedUrlTask.kt:43]: Let's add time - I think job be very slow if handle one by one

**Pull request merged**